frontend_lib/src/appContentFactory.js

```diff
@@ -1052,12 +1053,13 @@ export function appContentFactory (WrappedComponent) {
           await getSpaceMemberList(this.apiUrl, workspaceId)
         )
 
+        const includesKeyword = stringIncludes(keyword)
```

Comment:
I've just remembered that on frontend/ we have lodash library available
And, in lodash, there is _.includes that does the same thing as stringIncludes ([lodash _.includes doc](https://lodash.com/docs/4.17.15#includes))

It's generally better to use helper functions from vendors libraries that already are in the project because:
- it avoid adding new code
- the function have better documentation and examples
- the function is more tested
- other developers tends to be more familiars with it

I've checked the code base and it seems only frontend_app_workspace_advanced/ is using stringIncludes without having lodash in its package.json.
And that is making me think about adding lodash to frontend_vendors

And last comment about it, stringIncludes isn't unit tested

Conclusion:
- Quick solution: Add unit test to stringIncludes
- Middle solution: Refactor stringIncludes usage with _.includes and add lodash to frontend_app_workspace_advanced/
- Long solution: Refactor stringIncludes usage with _.includes, add lodash to frontend_vendors and remove lodash from every other package.json
- Alternative solution: change the body of stringIncludes to use _.includes. It would avoid having to change lodash imports and stringIncludes usages but will still require unit tests

Whether the quick, middle or the long solution is chosen, I'd like one of them to be implemented
I'm open to discuss which one to choose

Side note: that doesn't take away the fact that your implementation of stringIncludes is nice

Reply:
as seen together, we will implement the quick solution because judging by the usage of your function, it makes sense.
Also since this might might during the content list refactor, let's not spend a lot of time on it for now.